### PR TITLE
Visual Studio Code extension manifest fields

### PR DESCRIPTION
**Pull request by PatMyron**
https://code.visualstudio.com/api/references/extension-manifest

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,37 @@
 {
     "name": "vscode-cfn-lint",
-    "description": "CloudFormation linter",
+    "displayName": "CloudFormation Linter",
+    "description": "AWS CloudFormation Linter",
```
Comment (miparnisari) [inline package.json]: It says [here](https://code.visualstudio.com/api/references/extension-manifest#marketplace-presentation-tips):  "Provide a good display name and description. This is important for the Marketplace and in product displays. These strings are also used for text search in VS Code and having relevant keywords will help a lot."

Maybe put something more explicit, like "Linter for AWS Cloudformation templates, both JSON and YAML"?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
+    "icon": "logo.png",
+    "keywords": [
```
Comment (miparnisari) [inline package.json]: It says [here](https://code.visualstudio.com/api/references/extension-manifest#fields) that you shouldn't put more than 5 keywords

Comment (PatMyron) [reply package.json]: `"This list is currently limited to 5 keywords."` doesn't seem accurate:

https://github.com/dsteenman/cloudformation-yaml-snippets/blob/aeca3a451b2ae542decffc87fbacba5c0a973756/package.json#L28-L42

https://marketplace.visualstudio.com/items?itemName=dsteenman.cloudformation-yaml-snippets

Comment (miparnisari) [reply package.json]: Hmm then maybe put the most important 5 at the top just in case 🤷

**Pull request merged**